Sming/Arch/Esp32/Platform/StationImpl.cpp

```diff
@@ -15,10 +15,28 @@
 #include <esp_netif.h>
 #include <esp_event.h>
 
+#ifdef ENABLE_WPS
+#include <esp_wps.h>
+
+/*
+ * Information only required during WPS negotiation
+ */
+struct StationImpl::WpsConfig {
+	static constexpr unsigned timeoutMs{60000};
+	static constexpr unsigned maxRetryAttempts{5};
+	WPSConfigDelegate callback;
+	wifi_event_sta_wps_er_success_t creds;
+	uint8_t numRetries;
+	uint8_t credIndex;
+	bool ignoreDisconnects;
+};
+
+#endif
+
 static StationImpl station;
-StationClass& WifiStation = station;
+static esp_netif_t* stationNetworkInterface;
```

Comment:
Can you set the initial value of the pointer to be `nullptr`? 
```
static esp_netif_t* stationNetworkInterface = nullptr;
```

Reply:
Not necessary as C++ standard default-initialises globals and statics. But these shouldn't be global anyway so have moved them into respective classes.